### ISSUE TITLE
Move manim/container/container.py to manim/container.py

### ISSUE THREAD

**Issue by leotrs** [ManimCommunity/manim#288]
The subfolders inside the main source folder (`manim/`) is usually meant to either contain subpackages or serve organizational purposes. The case of the `manim/container/` folder serves neither of these, as `manim/container/container.py` is not a subpackage (only a module), and creating a whole folder for just one file (and a short one, at that), seems to yield very little organizational gain.

Basically, I think putting `container.py` in its own folder is unnecessary. I suggest we move `manim/container/container.py` to just `manim/container.py`. This will have no side effects facing the user.

**Comment by huguesdevimeux** [comment]
No opposition here.

**Comment by naveen521kk** [comment]
I think this would be good.
